bokeh/command/subcommands/info.py

```diff
@@ -72,3 +73,5 @@ def invoke(self, args):
             print("IPython version     :  %s" % ipy_version)
             print("Bokeh version       :  %s" % __version__)
             print("BokehJS static path :  %s" % settings.bokehjsdir())
+            print("node.js version     :  %s" % nodejs_version() or "(not installed)")
+            print("npm version         :  %s" % npmjs_version() or "(not installed)")
```

Comment:
can you make the wording consistent with `ipy_version` i.e. either `Not installed` or `(not installed)` Either is fine with me.